### PR TITLE
Feature/notebook

### PR DESCRIPTION
**Pull request by cdixson-ds**
Added the load.py file to connect to the 846 api, convert to df, and download as a csv. Experimented locally with changing out the data.csv and df_846.csv files, DS app routes were still functioning on my machine. Commented out my changes to the utilities.py file. 

Rubric: [ https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da ]

- [X ] - Which user story does this PR fall under, why is it necessary? 
Providing updated data to the user
- [ X] - Commit names descriptive of changes made
- [X ] - PR scoped around a specific issue
- [ ] - Peer reviewed (reviewer checks this box after having read code / leaving review comments)
- [X ] - Remove dead code(console logs, superfluous comments) 
- [ ] - Code follows good practices (concerns kept separate as per OOP)

### Feature/Issue
[remove placeholder text and write info about PR here, user story & feature built]

### Notes
[misc details go here]

### CODE REVIEW

File: project/requirements.txt [hunk 1]
```diff
@@ -4,6 +4,10 @@ fastapi==0.61.1
 jinja2==2.11.2
 pandas==1.1.3
 uvicorn==0.12.2
+<<<<<<< HEAD
```
Comment (BrokenShell) [inline project/requirements.txt:7]: What are these changes about? "<<<<< HEAD" etc.

File: project/load.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+soup = BeautifulSoup(response.data, "html.parser")
+
+json_846 = json.loads(soup.text)
+incidents = json_846['data']
+
+
+df_846 = pd.DataFrame(incidents)
+
+df_846.to_csv('df_846.csv')
+
```
Comment (BrokenShell) [inline project/load.py:22]: Should only have one blank line at the end of the file. PEP8

File: project/app/utilities.py [hunk 3]
```diff
@@ -5,7 +5,17 @@
 
 def get_data():
     """ Sets up the data for api filtering """
+<<<<<<< HEAD
```
Comment (BrokenShell) [inline project/app/utilities.py:8]: What is this: "<<< HEAD" etc.

File: project/app/__init__.py [hunk 4]
```diff
@@ -1 +1 @@
-from app.main import api
+from app.main import api
```
Comment (BrokenShell) [inline project/app/__init__.py:1]: I'm not sure why this shows as a change.

File: project/app/routes/reports.py [hunk 5]
```diff
@@ -39,3 +40,5 @@ async def full_report():
     returns a list of reports as JSON objects """
     result = DF.to_json(orient="records")
     return json.loads(result)
+
```
Comment (BrokenShell) [inline project/app/routes/reports.py:43]: Should only have one blank line at the end of the file. PEP8

**Comment by itshui3** [comment]
Good job spot-checking, keep helping each other maintain a high code quality =)

**Comment by cdixson-ds** [comment]
extraneous text was left in from VScode, cleaning up and making a new pull request

**Pull request closed without merging**